.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 build/
 .DS_Store
+.env.*
```

Comment:
I think I've typically seen it where the local/ dev version of the `.env` file is not gitignored, which helps make onboarding easier if you need to re-clone a repo (since you can just clone the non-gitignored one and change the filename)

Reply:
Good idea. I'll exclude it here.